src/components/typography/eventmetadata.js

```diff
@@ -0,0 +1,26 @@
+import React, { Fragment } from "react";
```

Comment:
this component is only used on the event template, so i think it would be better to define it there. it would also be ideal if your new component could inherit any properties from the existing Meta component for stylistic consistency. you could use the Meta component within your titled meta component with something like:
```
const TitledMeta = ({ children, title }) => {
  if (!title) {
    return <Meta>{ children }</Meta>
  }
  return (
    <Meta>
      <strong>{ title }:</strong>
      { children }
    </Meta>
  )

}
```
we don't want any Typography components looking one way in one place and another way in another place. generally, their styling should be changed either everywhere or nowhere.

Reply:
..or the Meta component itself could also accept an optional title prop.